src/Markup/Avalonia.Markup.Xaml/Avalonia.Markup.Xaml.csproj

```diff
@@ -74,6 +73,9 @@
         <Compile Include="PortableXaml\portable.xaml.github\src\Portable.Xaml\**\*.cs" Exclude="PortableXaml\portable.xaml.github\src\Portable.Xaml\Assembly\**\*.cs" />
         <Compile Remove="**\UriTypeConverter.cs" />
     </ItemGroup>
+    <ItemGroup>
+      <Compile Include="PortableXaml\portable.xaml.github\src\Portable.Xaml\Portable.Xaml.ComponentModel\UriTypeConverter.cs" />
```

Comment:
Remove ` <Compile Remove="**\UriTypeConverter.cs" />` instead of adding it here again